e2etests/stable_test.go

```diff
@@ -961,6 +961,31 @@ beta: {
 alpha -> beta: gamma {
 	style.font-color: green
 }
+`,
+		},
+		{
+			name: "latex",
+			script: `a: |latex
+\\Huge{\\frac{\\alpha g^2}{\\omega^5} e^{[ -0.74\\bigl\\{\\frac{\\omega U_\\omega 19.5}{g}\\bigr\\}^{\\!-4}\\,]}}
```

Comment:
why does it need to be `\\`?
I'd expect `\\` to render the actual `\` and not as a prefix to a latex command

Reply:
https://github.com/terrastruct/d2-docs/pull/15/files#diff-d23c28129c5e819c092f20bfcbf291203169d455eabed54a3498779040e06b20R36